roadmap/README.md

```diff
@@ -10,9 +10,81 @@ list for an overview of the initial roadmaps from the respective years.
 - [2021](./EF2021.md)
 - [2022](./EF2022.md)
 - [2023](./EF2023.md)
-- [2024](./EF2024.md) [ DRAFT, October 13 2023 ]
+- [2024](./EF2024.md)
 
-## Q4 2023
+
+## 2024 Recap
+
+In 2024 priorities shifted substantially over the year and some focus areas haven been downgraded substantially while others have been
+newly taken in. The following sections give a quick overview on the dynamics.
+
+### Classic EthereumJS Client Development (Downgraded)
+
+The EthereumJS [client](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client) is an indispensable research
+tool for EIP prototyping due to its highly modular structure. At the same time limits on classic usage became even more clear throughout
+the year, partly by the rise of even more performant L1 client alternatives like Reth. We have therefore substantially downgraded
+implementation efforts on things like SNAP sync, a flat database layout for state storage, a wider devp2p refactor or client 
+multithreading support, which initially had some stronger presence in our 2024 roadmap. Some of these might still be useful and followed
+upon on the sideline, though with less ressources applied.
+
+### Browser Support (Substantially Upgraded)
+
+JavaScript is THE programming language for the browser. Throughout the year it became clear that we had largely underappreciated one
+central browser-related topic and challenge around our libraries: the size of the bundles created from our libraries to be possible
+to be loaded in a web context. Especially for our EVM these were several 100KBs too large to get efficiently loaded within a 
+website context. We have therefore significantly ramped up efforts here, restructured a lot of code and removed unnecessary dependencies,
+culminating in our October/November 2024 breaking releases with bundle size reductions e.g. for our EVM of 60% and greater.
+
+Together with some other browser-related improvements these releases mark the first moment in time where the Ethereum L1 protocol
+stack is now fully ready to be used within web browsers without substantial limitations.
```

Comment:
I think I would expand this so we specially discuss how we restructured the code to make it tree-shakeable as that's the specific idea of a lot of our work this summer.

Reply:
Ok, I will put this a bit in, I am a bit sceptical though that people who will read this document will know what "tree shaking" is.

Reply:
Think I found a solid sentence to both explain and mention "tree shaking".